client/wfprev-war/src/main/angular/src/app/components/edit-project/project-details/evaluation-criteria/evaluation-criteria.component.ts

```diff
@@ -0,0 +1,104 @@
+import { CommonModule } from '@angular/common';
+import { Component, Input, OnChanges, SimpleChanges } from '@angular/core';
+import { MatDialog } from '@angular/material/dialog';
+import { MatExpansionModule } from '@angular/material/expansion';
+import { MatIconModule } from '@angular/material/icon';
+import { EvaluationCriteriaDialogComponent } from 'src/app/components/evaluation-criteria-dialog/evaluation-criteria-dialog.component';
+import { Project } from 'src/app/components/models';
+import { ExpansionIndicatorComponent } from 'src/app/components/shared/expansion-indicator/expansion-indicator.component';
+import { ProjectService } from 'src/app/services/project-services';
+
+@Component({
+  selector: 'wfprev-evaluation-criteria',
+  standalone: true,
+  imports: [ExpansionIndicatorComponent, MatExpansionModule, MatIconModule, CommonModule],
+  templateUrl: './evaluation-criteria.component.html',
+  styleUrl: './evaluation-criteria.component.scss'
+})
+export class EvaluationCriteriaComponent implements OnChanges {
+  @Input() project!: Project;
+
+  evaluationCriteriaSummaries: any[] = [];
```

Comment:
use an explicit type. No need to use `any`, especially within a list that seems to have a pretty definable type like evaluationCriteriaSummary